website/project/model.py

```diff
@@ -3782,6 +3782,30 @@ def to_json(self):
             "text": self.text
         }
 
+
+class DraftRegistrationLog(StoredObject):
+    """ Simple log to show status changes for DraftRegistrations
+
+    field - _id - primary key
+    field - date - date of the action took place
+    field - action - simple action to track what happened
+    """
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+    date = fields.DateTimeField(default=datetime.datetime.utcnow, index=True)
+    action = fields.StringField(index=True)
+
+    @classmethod
+    def add_log(cls, action):
```

Comment:
This doesn't seem necessary, and it's confusingly named; you don't add logs to a DraftRegistrationLog.  It would make more sense to have `DraftRegistration#add_status_log`.